script-library/awscli-debian.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
+# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/awscli.md
+# Maintainer: The VS Code and Codespaces Teams
+#
+# Syntax: ./awscli-debian.sh [AWS CLI version]
+
+set -e
+
+AWSCLI_VERSION=${1:-"latest"}
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    exit 1
+fi
+
+verify_aws_cli_gpg_signature() {
+    local filePath=$1
+    local sigFilePath=$2
+
+    get_common_setting AWSCLI_GPG_KEY
+    get_common_setting AWSCLI_GPG_KEY_MATERIAL
+    local awsCliPublicKeyFile=aws-cli-public-key.pem
+    echo "${AWSCLI_GPG_KEY_MATERIAL}" > "${awsCliPublicKeyFile}"
+    gpg --quiet --import "${awsCliPublicKeyFile}"
+    (
+    ) > "${awsCliPublicKeyFile}"
+    gpg --quiet --import "${awsCliPublicKeyFile}"
```

Comment:
I think this is left over content - I'm getting an error from here when running the script.

Reply:
Oops. 😅 Fixed in eb14d6934bf0022eef3791f94e251e734dae9c69.